### ISSUE TITLE
the input device is not a TTY

### ISSUE THREAD

**Issue by sigeshuo1** [farcasterxyz/hub-monorepo#2115]
**What is the bug?**
[A concise, high level description of the bug and how it affects you](https://github.com/sigeshuo1/hub-monorepo/blob/main/scripts/hubble.sh)

There is a problem with docker compose run when this script is executed non-interactively

e.g.
```shell
curl -s https://github.com/sigeshuo1/hub-monorepo/blob/main/scripts/hubble.sh | bash
```

When run in the above manner, Docker reports the following exception message:

```shell
the input device is not a TTY
```

**How can it be reproduced? (optional)**

Will the command line:
$COMPOSE_CMD run hubble yarn identity create
Modify to:
$COMPOSE_CMD run -T hubble yarn identity create

**Additional context (optional)**

**Comment by sds** [comment]
Hey @sigeshuo1, thanks for the report. 

It's not entirely clear to me why you would run the `hubble.sh` script this way, given we recommend you install via (see [instructions](https://www.thehubble.xyz/intro/install.html#install-via-script)):

```bash
curl -sSL https://download.thehubble.xyz/bootstrap.sh | bash
```

Can you provide more context? Otherwise we're inclined to close this as not necessary. Thanks!

**Comment by sds** [comment]
Going to close as not necessary, but if we're missing something do let us know. Thanks!

Closes #2116.

**Comment by sigeshuo1** [comment]
> 嘿，谢谢你的报告。
> 
> 我不完全清楚为什么要以这种方式运行脚本，因为我们建议您通过以下方式安装（请参阅[说明](https://www.thehubble.xyz/intro/install.html#install-via-script)）：`hubble.sh`
> 
> ```shell
> curl -sSL https://download.thehubble.xyz/bootstrap.sh | bash
> ```
> 
> 你能提供更多的背景信息吗？否则，我们倾向于关闭它，因为没有必要。谢谢！



> Hey @sigeshuo1, thanks for the report.
> 
> It's not entirely clear to me why you would run the `hubble.sh` script this way, given we recommend you install via (see [instructions](https://www.thehubble.xyz/intro/install.html#install-via-script)):
> 
> ```shell
> curl -sSL https://download.thehubble.xyz/bootstrap.sh | bash
> ```
> 
> Can you provide more context? Otherwise we're inclined to close this as not necessary. Thanks!

This makes it easier to install without having to download this shell file or go to chmod to set file permissions.

This is a best practice for shell execution!

The same is true for, for example, the rustup installer.
